protected/models/cron/cron.form.php

```diff
@@ -84,46 +57,36 @@ private function _getFormFields(
         return [$action, $interval, $isActive];
     }
 
-    /**
-     * Validate Data From Form
-     *
-     * @param string $action   Cron Job Action
-     * @param int    $interval Cron Job Execution Interval
-     * @param bool   $isActive Is Cron Job Active
-     *
-     * @return array Result Of Form Data Validation
-     */
-    private function _validateFormFields(
-        string $action   = '',
-        int    $interval = -1,
-        bool   $isActive = FALSE
-    ) : array
+    protected function _validateFormFields(
+        ?string $action   = null,
+        ?int    $interval = null,
+        bool    $isActive = false
```

Comment:
Codacy found an issue: [Variable "$isActive" is not in valid snake_case format, try "$is_active"](https://app.codacy.com/manual/common/buryak/pullRequest?prid=5800482)